### DISCUSSION TITLE
SEND AS ANY

### DISCUSSION THREAD

**Discussion by YosefAdPro** [postalserver/postal#1544] [Help with using Postal]
Hi,

I'm using postal for the first time and it looks great!

I want to ask about sending from any address.
I realized that in order to implement this, one needs to enable ``ALLOW SENDER HEADER`` And also ``SEND AS ANY``.

The problem is that we do not always have the option to add HEADER, when using postal as a relay.
Is there any way we can add HEADER on the postal side to outgoing messages or any other way?

Thanks.

**Comment by willpower232** [comment]
Unfortunately not. Postal wasn't intended to be a relay so you might find more success if you rearrange how you send emails a bit.

**Reply by YosefAdPro** [reply]
Thanks for your response.

> Postal wasn't intended to be a relay

I'm not sure I understood you. We connect applications and systems via SMTP/API (currently using Mailgun) to send mail.

So what is postal for?

**Reply by willpower232** [reply]
If you're sending emails via mailgun, Postal can replace that but you if you want to customise the email then it has to be in your applications and systems.
